test/csuite/wt10897_compact_quick_interrupt/main.c

```diff
@@ -194,29 +197,34 @@ main(int argc, char *argv[])
      * At this point, check that compact does not have any meaningful work to do.
      */
     skipped_compaction = false;
-    testutil_check(session->compact(session, uri, NULL));
+    testutil_check(session->compact(session, TABLE_URI, NULL));
     testutil_assert(skipped_compaction);
 
     /*
      * Now populate the table a lot more, make some space, and then see if we can interrupt the
-     * compaction quickly.
+     * compaction quickly - even before the compaction can get any work done.
      */
     populate(session, NUM_RECORDS1, NUM_RECORDS1 + NUM_RECORDS2);
     testutil_check(session->checkpoint(session, NULL));
     remove_records(session, 0, NUM_RECORDS1 + NUM_RECORDS2 / 2);
 
-    (void)time(&start);
     interrupted_compaction = false;
     skipped_compaction = false;
-    ret = session->compact(session, uri, NULL);
-    (void)time(&end);
+    ret = session->compact(session, TABLE_URI, NULL);
 
     testutil_assert(ret == WT_ERROR);
     testutil_assert(interrupted_compaction);
     testutil_assert(!skipped_compaction);
 
-    /* This should be more than enough time. */
-    testutil_assert(difftime(end, start) < 3 /* seconds */);
+    /*
+     * Check that we didn't get any work done.
+     */
```

Comment:
This can be a single-line comment. Likewise a couple places earlier in `main()`.